_posts/2015-12-15-how-bot-named-dolores-landingham-transformed-18fs-onboarding.md

```diff
@@ -0,0 +1,286 @@
+---
+title: "How a bot named Dolores Landingham transformed 18F’s onboarding"
+date: 2015-12-15
+authors:
+- jessie
+tags:
+- onboarding
+- how we work
+excerpt: "Our coworkers are very, very good at documenting the things they learn in Slack, our chat program, because it’s part of their daily workflow. So I tried an experiment: I asked my 18F coworkers to tag messages that every new 18F employee should know with the :evergreen_tree: emoji."
```

Comment:
It is, this is the part we use to generate the snippet that gets shared with Facebook and other open graph adopters. Reminds me, though, I should check the rendered source to make sure the tree gets generated in that section.